packages/css-library/dist/tokens/css/variables.css

```diff
@@ -22,32 +22,36 @@
   --color-primary-alt-darkest: #07648d;
   --color-primary-alt-light: #97d4ea;
   --color-primary-alt-lightest: #e1f3f8;
-  --color-secondary: #e31c3d;
-  --color-secondary-dark: #cd2026;
-  --color-secondary-darkest: #981b1e;
-  --color-secondary-light: #e59393;
-  --color-secondary-lightest: #f9dede;
-  --color-gray: #5b616b;
-  --color-gray-dark: #323a45;
+  --color-secondary: #d83933;
+  --color-secondary-dark: #b50909;
+  --color-secondary-darkest: #8b0a03;
+  --color-secondary-light: #f2938c;
+  --color-secondary-lightest: #f8dfe2;
+  --color-gray: #565c65;
+  --color-gray-dark: #3d4551;
   --color-gray-medium: #757575;
-  --color-gray-light: #aeb0b5;
-  --color-gray-light-alt: #eeeeee;
-  --color-gray-lighter: #d6d7d9;
-  --color-gray-lightest: #f1f1f1;
-  --color-gray-warm-dark: #494440;
-  --color-gray-warm-light: #e4e2e0;
-  --color-gray-cool-light: #dce4ef;
-  --color-green: #2e8540;
-  --color-green-dark: #1e5f2f;
-  --color-green-darker: #195c27;
-  --color-green-darkest: #0f3f1f;
-  --color-green-light: #7dbf7c;
-  --color-green-lighter: #94bfa2;
-  --color-green-lightest: #e7f4e4;
-  --color-gold: #fdb81e;
-  --color-gold-light: #f9c642;
-  --color-gold-lighter: #fad980;
-  --color-gold-lightest: #fff1d2;
+  --color-gray-light: #a9aeb1;
+  --color-gray-light-alt: #edeff0;
+  --color-gray-lighter: #dfe1e2;
+  --color-gray-lightest: #f0f0f0;
+  --color-gray-warm-dark: #454540;
+  --color-gray-warm-light: #e6e6e2;
+  --color-gray-cool-light: #e1e7f1;
+  --color-green: #008817;
+  --color-green-dark: #216e1f;
+  --color-green-darker: #154c21;
+  --color-green-light: #5e9f69;
+  --color-green-lighter: #b4d0b9;
+  --color-green-lightest: #ecf3ec;
+  --color-gold: #ffbe2e;
+  --color-gold-light: #face00;
+  --color-gold-lighter: #ffe396;
+  --color-gold-lightest: #fef0c8;
+  --color-gold-darker: #936f38;
+  --color-blue-cool: #345d96;
+  --color-blue-cool-light: #4a77b4;
+  --color-blue-cool-lighter: #98afd2;
+  --color-blue-cool-lightest: #e6e6e2;
   --v3-color-error-dark: #b50909;
   --v3-color-base-darkest: #1b1b1b;
```

Comment:
This might be related to your updates here or maybe not but do you think we can finally get rid of these temporary "v3 tokens" and instead use the actual USWDS token that maps to those hex values now?

```
 --v3-color-error-dark: #b50909;
 --v3-color-base-darkest: #1b1b1b;
```

Reply:
I went ahead and removed them and mapped any usage of them to variables with the same hex values.